conf/configuration.go

```diff
@@ -25,7 +25,7 @@ type nd struct {
 	IndexGroups     string `default:"A B C D E F G H I J K L M N O P Q R S T U V W X-Z(XYZ) [Unknown]([)"`
 
 	TranscodingCacheSize string `default:"100MB"` // in MB
-	ProbeCommand         string `default:"ffmpeg -i %s -f ffmetadata"`
+	ProbeCommand         []string `default:"[\"ffmpeg\", \"-\", \"ffmetadata\", \"-i\"]"`
```

Comment:
I don't think we should change this. `ProbeCommand` is part of the `conf` package so it can be potentially overridden. I don't think think `koding/multiconfig` (the lib that handles config loading) handles arrays, specially from environment vars. Did you test it?

Also, you left a `-i` here, but add another in the first append after this. it will cause the command to be:
`ffmpeg -ffmetadata -i -i ....`